v0/destinations/marketo_bulk_upload/poll.js

```diff
@@ -16,19 +16,18 @@ const getPollStatus = async event => {
   // To see the status of the import job polling is done
   // DOC: https://developers.marketo.com/rest-api/bulk-import/bulk-lead-import/#polling_job_status
   const requestOptions = {
-    url: `https://${munchkinId}.mktorest.com/bulk/v1/leads/batch/${event.importId}.json`,
-    method: "get",
     headers: {
       "Content-Type": "application/json",
       Authorization: `Bearer ${accessToken}`
     }
   };
+  const pollUrl = `https://${munchkinId}.mktorest.com/bulk/v1/leads/batch/${event.importId}.json`;
   const startTime = Date.now();
-  const resp = await httpSend(requestOptions);
+  const resp = await httpPOST(pollUrl, requestOptions);
   const endTime = Date.now();
   const requestTime = endTime - startTime;
   if (resp.success) {
-    if (resp.response && resp.response.data.success) {
+    if (resp.response && resp.response.data.data.success) {
```

Comment:
Does poll Status response has data inside response data? Did we check actual response from Marketo?